### DISCUSSION TITLE
MP4 conversion fails

### DISCUSSION THREAD

**Discussion by Typhooteaman** [sircharlo/meeting-media-manager#66] [Q&A] [answered]
Hi
Just installed the new update to version 0.8.98. Downloads work Ok, and the standard files are all there, but the mp4 conversion stops at the first file and doesn't move on. This is on an M1 Mac. Reverted back to 0.8.96 and mp4 conversion works Ok again.
Thanks for any help
Rob

**Comment by sircharlo** [comment]
Hi there @Typhooteaman ! Thanks for letting me know. What's displayed in the Console tab if you enable developer tools by hitting Command+Option+I?

**Comment by Typhooteaman** [comment, marked as answer]
Hi

This is the script
media-updater.js:423 Uncaught TypeError: Cannot read property 'streams' of undefined
    at media-updater.js:423
    at handleCallback (/Applications/JW Mee…/lib/ffprobe.js:106)
    at /Applications/JW Mee…/lib/ffprobe.js:145
    at /Applications/JW Mee…capabilities.js:194
    at wrapper (/Applications/JW Mee…c/dist/async.js:268)
    at next (/Applications/JW Mee…/dist/async.js:4582)
    at /Applications/JW Mee…c/dist/async.js:321
    at /Applications/JW Mee…capabilities.js:183
    at suppressedCallback (fs.js:227)
    at FSReqCallback.oncomplete (fs.js:171)

Screenshot

**Reply by sircharlo** [reply]
Thanks for your quick reply! Fixed in 0.8.99; should be released within 15 minutes

**Comment by Typhooteaman** [comment]
Hi

Just downloaded and installed, now works great again, thank you
